src/index.html

```diff
@@ -1,13 +1,240 @@
+            you know where your bike is and can track it anytime.
+          </p>
+        </article>
+        <article class="article">
+          <div class="pictures">
+            <img src="./images/photos/accessories3.jpg"
+                 class="pictures__item"
+                 alt="mobile phone with active appsense" />
+            <img src="./images/photos/accessories4.jpg"
+                 class="pictures__item"
+                 alt="holder for attaching to the handlebar" />
+          </div>
+          <h3 class="article--name">Range & Integrated lights</h3>
+          <p class="article--description"> The removable battery has up to 70km battery autonomy and weighs only 2.4 kg.
+            Lights integrated into the frame give you always-on visibility day and night.
+          </p>
+        </article>
+        <article class="article">
+          <div class="pictures">
+            <img src="./images/photos/accessories5.jpg"
+                 class="pictures__item"
+                 alt="mobile phone with active appsense" />
+            <img src="./images/photos/accessories6.jpg"
+                 class="pictures__item"
+                 alt="holder for attaching to the handlebar" />
+          </div>
+          <h3 class="article--name">Hydraulic disc brakes & Lightweight</h3>
+          <p class="article--description"> Brakes with total stopping power the second you make contact. The removable
+            battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you
+            always-on visibility day and night.
+          </p>
+        </article>
+        <a href="#"
+           class="button">Explore</a>
+      </section>
+
+      <section class="section">
+        <h2 class="title title--section title--section-contact">Contact us</h2>
+        <form class="form">
+          <div class="form__item">
+            <label for="name"
+                   class="form__label">Name</label>
+            <input id="name"
+                   class="form__input"
+                   type="text"
+                   placeholder="Name" />
+          </div>
+          <div class="form__item">
+            <label for="email"
+                   class="form__label">Email</label>
+            <input id="email"
+                   class="form__input"
+                   type="email"
+                   placeholder="Email" />
+          </div>
+          <div class="form__item">
+            <label for="message"
+                   class="form__label">Message</label>
+            <textarea id="message"
+                      class="form__textarea"
+                      placeholder="Message">
+            </textarea>
+          </div>
+          <button class="button button--primary">Send</button>
+        </form>
+        <address class="address">
+          <h4 class="address__label">Phone</h4>
+          <p class="address__content">+1 234 5555-55-55</p>
```

Comment:
it should be `<a>` with possibility to call

Reply:
amended to <a>